app/src/main/java/com/example/tasky/feature_agenda/domain/use_case/DeleteReminder.kt

```diff
@@ -19,9 +22,9 @@ class DeleteReminder(
         return when (result) {
             is Resource.Error -> {
                 when (result.errorType) {
-                    ErrorType.HTTP -> Result.Error(result.message ?: "Unknown Error")
+                    HTTP -> Result.Error(result.message ?: "Unknown Error")
 
-                    ErrorType.IO -> {
+                    IO -> {
```

Comment:
Why will the item only scheduled to be synced in case of an IO error? Depending on the HTTP error, it should also be synced there. Imagine, the server has an error and responds with 500